### ISSUE TITLE
Tracking issue

### ISSUE THREAD

**Issue by In-line** [s1lentq/ReGameDLL_CS#185]
ValveSoftware/halflife#278

**Comment by simeonniko** [comment]
Is this already fixed? https://forums.alliedmods.net/showpost.php?p=1463034&postcount=15

**Comment by Daniele386** [comment]
No. 18 seems to be [dreamstalker/rehlds#558](https://github.com/dreamstalker/rehlds/issues/558), maybe other can confirm this too

**Comment by Daniele386** [comment]
Maybe some Fixes from https://forums.alliedmods.net/showthread.php?t=171831 can be implemented?

**Comment by Daniele386** [comment]
I can confirm/reproduce No. 23 

I can confirm/reproduce **No. 13 only on original gamedll**; it seems to be fixed on ReHLDS/ReGameDLL (I had AMXX disabled, so it's not AMXX related; I used dual berretas to reload as it takes very long)

**Comment by fred0r** [comment]
think i didnt see it on the list - would be nice if flashes which explode behind players doesnt (under certain circumstances) fullflash them.
this amxmodx already seems to fix this:
https://forums.alliedmods.net/showpost.php?p=930803&postcount=42
